deployments/dlab/config/common.yaml

```diff
@@ -12,31 +12,90 @@ jupyterhub:
     chp:
       nodeSelector:
         hub.jupyter.org/pool-name: core-pool
-    traefik:
-      nodeSelector:
-        hub.jupyter.org/pool-name: core-pool
 
   hub:
     nodeSelector:
       hub.jupyter.org/pool-name: core-pool
     config:
       Authenticator:
         admin_users:
-            # infrastructure
-            - sknapp
-            - rylo
-            - yuvipanda
-            - felder
-            - balajialwar
-            # List of other admin users
-            - rbarreto
-            - pratik.sachdeva
-            - aculich
-            - cheng.ren
-            - emily_grabowski
-            - klganter
-            - cvacano
+          # infrastructure
+          - sknapp
+          - rylo
+          - yuvipanda
```

Comment:
lets remove yuvi from this and all future PRs.

Reply:
He'll need to be removed from the [cookiecutter](https://github.com/berkeley-dsep-infra/datahub/blob/staging/deployments/template/%7B%7Bcookiecutter.hub_name%7D%7D/config/common.yaml).